ecl/hql/hqlgram2.cpp

```diff
@@ -1013,7 +1013,11 @@ IHqlExpression * HqlGram::processEmbedBody(const attribute & errpos, IHqlExpress
         }
     }
     else
+    {
+        if (attribs && attribs->queryName()==activityAtom)
```

Comment:
This test is wrong - attribs is a comma list - should be using queryAttributeInList